packages/node/src/app/analytics-node.ts

```diff
@@ -39,294 +57,263 @@ export interface AnalyticsSettings {
   writeKey: string
   timeout?: number
   plugins?: CorePlugin[]
+  integrations?: Integrations
+  retryQueue?: boolean
```

Comment:
🤔 I think eventually we can get rid of these, since server-side destinations do not have a notion of `integrations`. Also we are not implementing any retry mechanism? are we?